conf/init/ol-solr-updater.conf

```diff
@@ -6,5 +6,6 @@ stop on runlevel [!2345]
 
 chdir /openlibrary
 respawn
```

Comment:
Should the first respawn be removed (replaced with the second `respawn limit 10 5`)? Or should both be present